exammode/models.py

```diff
@@ -0,0 +1,120 @@
+from django.db import models
+from django.utils import timezone
+
+from course.models import CourseInstance, CourseModule
+from exercise.exercise_models import LearningObject
+from userprofile.models import UserProfile
+
+# Create your models here.
+
+
+class ExamSessionManager(models.Manager):
+
+    def get_queryset(self):
+        initial_queryset = super().get_queryset()
+        queryset = [q for q in initial_queryset if (
+            q.can_start <= timezone.now())]
+
+        return queryset
+
+    def is_active(self):
+        initial_queryset = super().get_queryset()
+        queryset = [q for q in initial_queryset if (q.can_start <= timezone.now(
+        ) and (timezone.now() <= q.can_start + timezone.timedelta(hours=q.duration)))]
+
+        return queryset
+
+
+class ExamSession(models.Model):
+    """
+    Represents one instance of a course exam
+    """
+
+    course_instance = models.ForeignKey(
+        CourseInstance, on_delete=models.CASCADE)
+    exam_module = models.ForeignKey(
+        CourseModule, on_delete=models.CASCADE, null=True)
+    can_start = models.DateTimeField(editable=True, default=timezone.now)
+    duration = models.IntegerField()
+    start_time_actual = models.DateTimeField(
```

Comment:
Who sets `start_time_actual`? I suppose it can't be set beforehand when the teacher prepares the exam and the exam session in A+. Is the field supposed to be empty at first when the exam session is created?

Reply:
Unused field for unimplemented feature. To be removed.

Reply:
Fixed